### ISSUE TITLE
'line_profiler' is a package and cannot be directly executed

### ISSUE THREAD

**Issue by donglixp** [rkern/line_profiler#176]
Command: `python -m line_profiler test.py.lprof`

Error messages:
```bash
/data/anaconda/envs/p36/bin/python: No module named line_profiler.__main__; 'line_profiler' is a package and cannot be directly executed
```

**Comment by rkern** [comment]
Please direct this to the new home: https://github.com/pyutils/line_profiler

That transformation into a package happened there.

**Comment by timbmg** [comment]
Facing the same and wanted to create the issue on the new repo. But I think you'd need to allow issues on that fork @rkern

**Comment by rkern** [comment]
I'm not related to that fork.

**Comment by jakevdp** [comment]
If anyone comes across this, the issue seems to be with the 3.0 release on January 6. If you ``pip install line_profiler~=2.0`` it should work as advertised.

I'd raise the issue on the new repo, but it doesn't appear to be accepting issues.

**Comment by rkern** [comment]
@Erotemic Can you allow issues on the new fork so these issues can be reported appropriately? Thank you.

**Comment by Erotemic** [comment]
Didn't realize forks didn't have issues enabled by default. I enabled it. 

@rkern I also invited you to the pyutils organization in case you want to continue to be involved with the package.

**Comment by Erotemic** [comment]
@donglixp @jakevdp @timbmg This issue should be resolved in the latest 3.0.1 release, please try it out and let me know if there are any issues. The pyutils fork can now accept issues, so please post them there.
